### DISCUSSION TITLE
docs/kubernetes-hardening-guidance/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/cloud-native-library#14] [Announcements]
# docs/kubernetes-hardening-guidance/

本指南译自美国国家安全局（NSA）于 2021 年 8 月发布的的 Kubernetes Hardening Guidance。

https://lib.jimmysong.io/kubernetes-hardening-guidance/

**Comment by rootsongjc** [comment]
在 2021 年 8 月我翻译这个报告的时候，感觉有的部分似乎还没有写完，比如“认证和授权”一节，我想应该会发布后续版本。

**Comment by wenyurush** [comment]
https://media.defense.gov/2022/Aug/29/2003066362/-1/-1/0/CTR_KUBERNETES_HARDENING_GUIDANCE_1.2_20220829.PDF

现在有新版本的了，考考更新么

**Reply by rootsongjc** [reply]
最近这个月比较忙，得比较下新版本与旧版本变更的范围和内容的多少，再预估下需要多长时间来再翻译。
